pom.xml

```diff
@@ -2335,7 +2335,7 @@ under the License.
 				<plugin>
 					<groupId>com.github.siom79.japicmp</groupId>
 					<artifactId>japicmp-maven-plugin</artifactId>
-					<version>0.17.1</version>
+					<version>0.18.5</version>
```

Comment:
This one is new!☝️The following error is shown by japicmp:
```
[ERROR] Failed to execute goal com.github.siom79.japicmp:japicmp-maven-plugin:0.17.1:cmp (default) on project flink-state-processor-api: There is at least one incompatibility: org.apache.flink.state.api.OperatorTransformation.bootstrapWith(org.apache.flink.api.java.DataSet,long):CLASS_GENERIC_TEMPLATE_CHANGED,org.apache.flink.state.api.OperatorTransformation.bootstrapWith(org.apache.flink.streaming.api.datastream.DataStream,long):CLASS_GENERIC_TEMPLATE_CHANGED -> [Help 1]
```

The problem is caused by https://github.com/siom79/japicmp/issues/368.
The issue behind can be solved the following ways:
* We increase the jacmp version where the mentioned bug has been resolved
* We add class to the whitelist

Reply:
Hi @gaborgsomogyi  How about bump this on master branch and backport it here?

Reply:
I'm fine with anything since this is ugly here. Separating the intentions is good in general so doing it.

Reply:
https://issues.apache.org/jira/browse/FLINK-37480

Reply:
When it's merged then we're going to have the possibility to remove this part.

Reply:
It's no more in this change.